everyvoice/wizard/dataset.py

```diff
@@ -684,6 +685,10 @@ def reload_filelist_data_as_dict(state):
                 state.get(StepNames.data_has_header_line_step, "yes") == "yes"
             ),
         )
+        state["filelist_data"] = []
+        for row in state["filelist_data_list"][1:]:
```

Comment:
What is dropped in `state["filelist_data_list"][0]`?
Why are we dropping it?

Reply:
The first element of `state["filelist_data_list"]` is the header

Reply:
Got it